src/payment/strategies/digitalriver/digitalriver-payment-strategy.ts

```diff
@@ -31,60 +34,57 @@ export default class DigitalRiverPaymentStrategy implements PaymentStrategy {
 
     async initialize(options: PaymentInitializeOptions): Promise<InternalCheckoutSelectors> {
         this._digitalRiverInitializeOptions = options.digitalriver;
+        const paymentMethod = this._store.getState().paymentMethods.getPaymentMethodOrThrow(options.methodId);
+        const { publicKey, paymentLanguage: locale } = paymentMethod.initializationData;
+        const digitalRiverWindow: DigitalRiverWindow = await this._digitalRiverScriptLoader.load();
+        const { containerId } = this._getDigitalRiverInitializeOptions();
 
-        const state = await this._store.dispatch(this._paymentMethodActionCreator.loadPaymentMethod(options.methodId));
-        const paymentMethod = state.paymentMethods.getPaymentMethodOrThrow(options.methodId);
-        let clientToken: DigitalRiverInitializeToken;
+        this._digitalRiverInstance = digitalRiverWindow.DigitalRiver;
```

Comment:
Rather than keeping a local copy of `digitalRiverInstance` in this class, I think it makes more sense to return the instance as part of `this._digitalRiverScriptLoader.load()`

If the script has already been loaded, then `this._digitalRiverScriptLoader.load()` can generate a new instance. Basically the change would be to move the variable `this._digitalRiverInstance` to DigitalRiverScriptLoader.